<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="up">
        <property name="text">
         <string>UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="down">
        <property name="text">
         <string>DOWN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="on">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restore">
        <property name="text">
         <string>RESTORE PREVIOUS </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="off">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="clock">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="elecOff">
      <property name="text">
       <string>ElectrodeOff</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="incPower">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="powerLevel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
;
border-color: rgb(0, 0, 0);
font: 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decPower">
        <property name="styleSheet">
         <string notr="true">
font: 20pt &quot;DejaVu Sans&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="chargeDev">
      <property name="text">
       <string>CHARGE DEVICE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="elecOn">
      <property name="text">
       <string>ElectrodeON</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;DejaVu Sans&quot;;
border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="treatment">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>TREATMENT:  NONE </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>on</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>off</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
